packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/About.2.doc.mdx

```diff
@@ -2,26 +2,120 @@ import { Notice } from "@cerner/terra-docs";
 import { Badge } from 'terra-grid/package.json?dev-site-package';
 
 import WorklistDataGridPropsTable from 'terra-data-grid/src/WorklistDataGrid?dev-site-props-table';
-
 import DefaultWorklistDataGrid from './Examples.3/DefaultWorklistDataGrid?dev-site-example';
-import WorklistDataGridColumnStates from './Examples.3/WorklistDataGridColumnStates?dev-site-example';
 
 <Badge />
 
 # Terra WorklistDataGrid
```

Comment:
Looks like this got missed, and in the Caution label, for Worklist Data Grid

Reply:
Updated